### ISSUE TITLE
Deploy process stuck on Waiting for the emulator

### ISSUE THREAD

**Issue by lzrdig** [xamarin/get-started#137]
Problem encountered on https://dotnet.microsoft.com/en-us/learn/xamarin/hello-world-tutorial/run
Operating System: windows

When deploying Android build to the emulated device the build process gets stuck on "Waiting for the emulator is ready...". When I check if Android's debug bridge can connect to the emulator it comes back with not able to start server on port 5037. I have checked all firewall settings and it still cannot connect. When using "Test_NetConnection localhost -Port 5037" I get connection refused. Any ideas as to how to get the sample come up onto the emulated device would be appreciated.

**Comment by jamesmontemagno** [comment]
i would maybe try to open up the android  adb command prompt (tools -> android)


adb kill-server
adb start-server

should reboot it

**Comment by lzrdig** [comment]
Well, it turns out that it is more complicated than that. I ended up trying the same process on three other PCs and it worked there. One of them was not connecting at the beginning, but then it started working after I uninstalled and then installed back the emulator.

On the machine that does not work, however, I found that adb daemon works fine if I use ports higher than 6000. But, I cannot use that for debug since VS insists on trying to use adb with port 5307.
One difference that I have on the machine with the issue for ADB is that I have both VS2022 and VS2019 installed.
I have not tried uninstalling one of them yet. I did try repair on either and it helped intermittently (too long to describe here). So, I am inclined to think that it does not like both VS IDEs. I am surprised however that netstat nor firewall settings do not show 5307 port being taken by anything else.
If I knew what config settings ADB uses for kicking off the daemon, then, perhaps I could see that the connection session if marked as being taken.

In any case, I will go ahead and ask the issue to be closed.

Thanks.
